MovieApp/Scenes/DetailsPage/DetailsComponentsView.swift

```diff
@@ -0,0 +1,398 @@
+                constant: Constants.ClockImage.topPadding
+            ),
+            clockImage.leadingAnchor.constraint(
+                equalTo: timeView.leadingAnchor,
+                constant: Constants.ClockImage.leadingPadding
+            )
+        ])
+    }
+    
+    private func setUpHourLabel() {
+        timeView.addSubview(hourLabel)
+        
+        NSLayoutConstraint.activate([
+            hourLabel.topAnchor.constraint(
+                equalTo: timeView.topAnchor,
+                constant: Constants.HourLabel.topPadding
+            ),
+            hourLabel.leadingAnchor.constraint(
+                equalTo: clockImage.trailingAnchor,
+                constant: Constants.HourLabel.leadingPadding
+            ),
+            hourLabel.trailingAnchor.constraint(
+                lessThanOrEqualTo: timeView.trailingAnchor,
+                constant: Constants.HourLabel.trailingPadding
+            ),
+            hourLabel.bottomAnchor.constraint(
+                equalTo: timeView.bottomAnchor,
+                constant: Constants.HourLabel.bottomPadding
+            )
+        ])
+    }
+    
+    //MARK: - Movie Year
+    private func setUpYearView() {
+        addSubview(yearView)
+        
+        NSLayoutConstraint.activate([
+            yearView.topAnchor.constraint(
+                equalTo: movieName.bottomAnchor,
+                constant: Constants.YearView.topPadding
+            ),
+            yearView.leadingAnchor.constraint(
+                equalTo: timeView.trailingAnchor,
+                constant: Constants.YearView.leadingPadding
+            ),
+            yearView.bottomAnchor.constraint(equalTo: bottomAnchor)
+        ])
+        yearView.translatesAutoresizingMaskIntoConstraints = false
+    }
+    
+    private func setUpYearLabel() {
+        yearView.addSubview(yearLabel)
+        
+        NSLayoutConstraint.activate([
+            yearLabel.topAnchor.constraint(
+                equalTo: yearView.topAnchor,
+                constant: Constants.YearLabel.topPadding
+            ),
+            yearLabel.leadingAnchor.constraint(
+                equalTo: yearView.leadingAnchor,
+                constant: Constants.YearLabel.leadingPadding
+            ),
+            yearLabel.trailingAnchor.constraint(
+                lessThanOrEqualTo: yearView.trailingAnchor,
+                constant: Constants.YearLabel.trailingPadding
+            ),
+            yearLabel.bottomAnchor.constraint(
+                equalTo: yearView.bottomAnchor,
+                constant: Constants.YearLabel.bottomPadding
+            )
+        ])
+    }
+}
+
+//MARK: - Constants
+private extension DetailsComponentsView {
+    enum Constants {
+        enum Title {
+            static let movieName = "The Atonement"
+            static let imdb = "7.9"
+            static let genre = "Romance"
+            static let hour = "1h 50m"
+            static let year = "2007"
+        }
+        enum imdbView {
```

Comment:
`enum` ისევე როგორც სხვა ტიპები, მაგალითად `struct` და `class` აუცილებლად დიდი ასოთი უნდა იწყებოდეს. 

ამასთან imdb-ში (და ზოგადად ნებისმიერ აბრეიატურაში) ან ყველა ასო დიდი უნდა იყოს, ან ყველა პატარა.

```suggestion
        enum IMDBView {
```